hedera-node/configuration/mainnet/settings.txt

```diff
@@ -26,3 +26,4 @@ useLoopbackIp,                                 false
 waitAtStartup,                                 false
 jasperDb.storagePath,                          /opt/hgcapp/services-hedera/HapiApp2.0/data/saved
 jasperDb.iteratorInputBufferBytes,             16777216
+jasperDb.indexRebuildingEnforced,              true
```

Comment:
Q: Should there be a bootstrap override for the mainnet setting `contracts.chainId=295` for genesis reconnect?

Reply:
We'll set this value in the _bootstrap.properties_ JAR on the release branch.

Reply:
Okay